MAINTAINERS.md

```diff
@@ -0,0 +1,77 @@
+# Maintainers Guide
```

Comment:
I think this document should be `CONTRIBUTING.md` and `MAINTAINERS` should be a separate file.